astroid/__pkginfo__.py

```diff
@@ -35,8 +35,12 @@ def has_environment_marker_range_operators_support():
 
 if has_environment_marker_range_operators_support():
     extras_require[':python_version<"3.4"'] = ['enum34', 'singledispatch']
-elif py_version < (3, 4):
-    install_requires.extend(['enum34', 'singledispatch'])
+    extras_require[':python_version<"3.2"'] = ['functools32']
```

Comment:
Sorry for being late to the party, I thought this was still work-in-progress.

Why don't we use https://pypi.python.org/pypi/backports.functools_lru_cache/ instead? this is what we use in pylint and it seems official, rather than functools32.